ParabolInc/parabol
# chore(kudos): add kudos record when adding emoji reaction

fix #9091

**How to test**

- Add kudos org level feature flag
- Use ❤️ reaction for sending kudos
- See snackbar appears

<img width="523" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/68ab7d19-da41-459d-bd3b-0243820cae91">


Comments

- Liking comment should send kudos
- Liking your own comment should not add kudos
- Liking anonymous comments should not add kudos
- Removing reaction should not have any effect

Reflections

- Liking anonymous reflections should not create kudos
- Liking non-anonymous reflections should create kudos


Standup response

- Liking standup response should create kudos
